jpype/_core.pyi

```diff
@@ -1,2 +0,0 @@
-class _JRuntime:
```

Comment:
was this ever used?

Reply:
I _think_ it only existed because we didn't want to add type annotations into the ``_core.py`` (possibly because we supported Python 2 still at the time). Now that we can inline the annotations, they are far better close to the actual implementation to ensure they remain correct.

Reply:
That makes sense. But I'd suggest to open another PR for that. Mixing multiple things makes it harder to track history.

Reply:
Thanks for your review, I agree completely. In this case, removing this file is fundamental to the PR, since it was influencing the tools to believe that the only interface available in ``_core`` was ``_JRuntime``.